MinecraftClient/Protocol/Handlers/Protocol16.cs

```diff
@@ -733,5 +733,23 @@ public static bool doPing(string host, int port, ref int protocolversion)
             }
             catch { return false; }
         }
+
+        // reinforce
+        public bool SendInteractEntityPacket(int EntityID, int type)
+        {
+            return true;
```

Comment:
Non-implemented methods should return false, also these addition could be moved to line 643 below SendClientSettings() and SendLocationUpdate() which are also not implemented.